### DISCUSSION TITLE
Will there be LTCT algs avaliable?

### DISCUSSION THREAD

**Discussion by gpjalocha** [nbwzx/blddb#7] [Ideas]
LTCT is a useful algset and it would be worth to have it there.

some spreadsheets to use from 

https://docs.google.com/spreadsheets/d/1h6OtATn_A6RBJ2s47XWe_Q2kYIqgBO0E/
https://docs.google.com/spreadsheets/d/1400SA2bapIsmbgMlkHFEMnFZ9YHYNJJxAygx6BH9Eys/edit?usp=sharing
https://docs.google.com/spreadsheets/d/1MSm0iavGr2NGP7m4RcIBbAWZplwU8eHH4RFqPymb1co/edit#gid=0

**Comment by nbwzx** [comment]
Thanks. However, unlike 3-style corner and 3-style edge algs, there are too few ltct spreadsheets to refer to, I'm not sure what a suitable display format is.

**Comment by nbwzx** [comment]
@gpjalocha 
I am sorry for the long delay to add LTCT algset.
Now I have released the LTCT & T2C module in BLDDB.
If you have any questions or suggestions, please let me know.
Close this discussion as resolved.
:)

**Comment by gpjalocha** [comment]
Splendid! Thank you very much!!
